packages/core/phase3/lib/pncUpdateRequestGenerators/normalDisposalGenerator/normalDisposalGenerator.test.ts

```diff
@@ -0,0 +1,240 @@
+import { isError } from "@moj-bichard7/common/types/Result"
+
+import type { Offence, OffenceReason, Result } from "../../../../types/AnnotatedHearingOutcome"
+import type { PncOperation } from "../../../../types/PncOperation"
+import type { Operation } from "../../../../types/PncUpdateDataset"
+import type NormalDisposalPncUpdateRequest from "../../../types/NormalDisposalPncUpdateRequest"
+
+import lookupOrganisationUnitByCode from "../../../../lib/dataLookup/lookupOrganisationUnitByCode"
+import generatePncUpdateDatasetFromOffenceList from "../../../../phase2/tests/fixtures/helpers/generatePncUpdateDatasetFromOffenceList"
+import ResultClass from "../../../../types/ResultClass"
+import normalDisposalGenerator from "./normalDisposalGenerator"
+
+jest.mock("../../../../lib/dataLookup/lookupOrganisationUnitByCode")
+const mockedLookupOrganisationUnitByCode = lookupOrganisationUnitByCode as jest.Mock
+
+const createPncUpdateDataset = () => {
+  const offence = {
+    OffenceCategory: "ZZ",
+    CriminalProsecutionReference: {
+      OffenceReason: {
+        __type: "NationalOffenceReason",
+        OffenceCode: {
+          __type: "NonMatchingOffenceCode",
+          ActOrSource: "RT",
+          Year: "88",
+          Reason: 191,
+          FullCode: "RT88191"
+        }
+      } as unknown as OffenceReason
+    },
+    AddedByTheCourt: true,
+    ActualOffenceStartDate: {
+      StartDate: new Date("2025-01-01")
+    },
+    Result: [
+      {
+        NextHearingDate: "2024-12-11T10:11:12.000Z",
+        NextResultSourceOrganisation: {
+          TopLevelCode: "B",
+          SecondLevelCode: "01",
+          ThirdLevelCode: "00",
+          BottomLevelCode: "00",
+          OrganisationUnitCode: "B000000"
+        },
+        PNCDisposalType: 2059,
+        PNCAdjudicationExists: true,
+        ResultClass: ResultClass.ADJOURNMENT_PRE_JUDGEMENT,
+        ResultQualifierVariable: [{ Code: "LE" }]
+      }
+    ] as Result[]
+  } as Offence
+
+  const pncUpdateDataset = generatePncUpdateDatasetFromOffenceList([offence])
```

Comment:
I don't think using this test helper is necessary as we end up just overwriting everything in the end. I think just create the object here and it should be easier to read too.

Reply:
Agreed. It looks more readable when constructing the AHO object in this test.